### ISSUE TITLE
Module parse failed: Cannot use 'import.meta' outside a module 

### ISSUE THREAD

**Issue by dillionverma** [vercel-labs/react-tweet#80]
<img width="1034" alt="image" src="https://github.com/vercel-labs/react-tweet/assets/16860528/01338693-5358-425e-bfff-8e471fbec547">

```
- error ./node_modules/react-tweet/node_modules/@babel/runtime/helpers/interopRequireDefault.js
Module parse failed: Cannot use 'import.meta' outside a module (39:16)
|                 // still a Refresh Boundary later.
|                 // @ts-ignore importMeta is replaced in the loader
>                 import.meta.webpackHot.accept();
|                 // This field is set when the previous version of this module was a
|                 // Refresh Boundary, letting us know we need to check for invalidation or


```

Getting the following error while running.

package.json:

```json

    "eslint-config-next": "^13.4.2",
    "next": "^13.4.2",
    "next-auth": "^4.22.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-tweet": "^2.0.1",

```

**Comment by lakbychance** [comment]
@lfades Hitting the same limitation. SWC is disabled for our project because we rely on babel for a library. I tried to create a reproduction online but couldn't. This is what I was trying -> https://codesandbox.io/p/sandbox/angry-robinson-sk8mrv?file=%2Fpackage.json%3A22%2C36. Basically just added a `.babelrc.js` similar to our current setup so that next disables SWC on detecting it but I can see atleast the loader of the Tweet component coming. Whereas, in our local dev, we are getting the same error like so :- 
<img width="1138" alt="Screenshot 2023-07-27 at 3 57 00 PM" src="https://github.com/vercel/react-tweet/assets/29139815/dfac583f-708a-4fef-b996-6c6ed3cba759">

Any clue as to what might be happening here ?

**Comment by lfades** [comment]
Looks like it's something with babel when used in Next.js. The package output of react-tweet does not use `import.meta` and all files are modules. Did you also add `reac-tweet` to [`transpilePackages`](https://react-tweet.vercel.app/next#troubleshooting)? although I'm not entirely sure if that one works with babel.

**Comment by lakbychance** [comment]
> Looks like it's something with babel when used in Next.js. The package output of react-tweet does not use `import.meta` and all files are modules. Did you also add `reac-tweet` to [`transpilePackages`](https://react-tweet.vercel.app/next#troubleshooting)? although I'm not entirely sure if that one works with babel.

Yup the `transpilePackages ` is already added to `next.config.js`

**Comment by jshwrnr** [comment]
Getting this issue as well.

**Comment by reabreu** [comment]
Hey everyone! I'm also getting this issue, does anyone have an ideia why this is happening?

**Comment by dillionverma** [comment]
So I resolved this for myself by using app router instead (Nextjs 13).

Not a feasible solution for everyone but just wanted to let people know

**Comment by reabreu** [comment]
Hey, thanks for the feedback!

**Comment by timbicker** [comment]
I solved the problem by adding 
```
"resolutions": {
  "@babel/runtime": "^7.24.6"
},
```
to my package.json.
As explained in [this issue](https://github.com/vercel/next.js/issues/35110#issuecomment-2131108641)

**Comment by Rajesh-Nagarajan-11** [comment]
Any solution for next 15 version ?? // @timbicker @lfades @reabreu @jshwrnr @dillionverma
